docs/Contributing.asciidoc

```diff
@@ -263,4 +263,4 @@ Authentication module is expected to return HASH:
 --------------------------------------------------------------------------------
 
 Authentication module is expected to create or update user entry in OpenQA database
-after user validation user. See included modules for inspiration.
\ No newline at end of file
+after user validation user. See included modules for inspiration.
```

Comment:
I think you were trying to fix something here, weren't you? The second 'user' is extra.